content/changelog/2024-02-09-console.md

```diff
@@ -0,0 +1,67 @@
+---
+description: pgvector 0.6.0, IPv6 is coming, prepared statement support for pooled connections, and more
+---
+
+### Support for pgvector 0.6.0
```

Comment:
we rolled it back. There are issues.

cc @sharnoff

Reply:
Thank you. Removed.